### ISSUE TITLE
Update color on completed?

### ISSUE THREAD

**Issue by Fisher26** [visionmedia/node-progress#178]
Say I do something like 

```
bar = new ProgressBar(':current/:total :bar :percent :elapseds', {
		total: files.length,
		complete: red,
	});
```

And then once I'm done, I'd run something like `bar.update({ complete: green });`, which re-renders the bar, but doesn't change the color. Am I missing something, or is this not supported?

**Comment by mearns** [comment]
I don't believe this is currently supported: the `update` method doesn't reconfigure the progress bar, it sets the percent complete (and optionally provides render tokens). You *could* manually set the config properties on the `ProgressBar`in the `callback`, and then force it to re-render, but there are some issues with this (namely, the bar is finalized before the callback, so if you call `render` again from the callback, it will render on a new line, instead of overwriting the existing line).

I've made some minor changes and added a `config` method in pull request #193 to support the requested feature, as well as provided an example in `examples/update-on-finish.js`.

**Comment by pbeocanin** [comment]
Yeah I don't think it's a huge deal, but it would be nice to visually indicate it's done, so I'm definitely going to try out the pull request directly, thank you!

**Comment by zhizhuxialiwen** [comment]
I think that it updates progress bar , then it updates color.
